adyen.php

```diff
@@ -679,7 +635,7 @@ public function hookPaymentOptions()
                 'paymentProcessUrl' => $this->context->link->getModuleLink($this->name, 'Payment', array(), true),
                 'threeDSProcessUrl' => $this->context->link->getModuleLink($this->name, 'ThreeDSProcess', array(), true),
                 'prestashop16' => false,
-                'loggedInUser' => !$this->context->customer->is_guest
+                'loggedInUser' => (int)!$this->context->customer->is_guest
```

Comment:
is this always a integer ? I could imagine this default value can be overwritten to be any value.

Reply:
The value here would be a boolean

Reply:
clear.